### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by twasyl**
The changes introduce terminology updates to replace `master` by `controller` and remaining `slave` by `agent`.
This is lead by [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: docs/troubleshooting.adoc [hunk 1]
```diff
@@ -341,16 +341,16 @@ Taken from https://support.quest.com/SolutionDetail.aspx?id=SOL86281
...
-If you are changing configuration of the jenkins it is possible that the agent tries to connect to old masters address.
+If you are changing configuration of the jenkins it is possible that the agent tries to connect to old controllers address.
 To fix this:
 
 . on the agent: stop jenkins windows service (if not already dead)
-. on the master: go to Jenkins > Manage Jenkins > Configure System, and
+. on the controller: go to Jenkins > Manage Jenkins > Configure System, and
 copy value from 'Jenkins URL' parameter
-. on the agent: edit jenkins-slave.xml and alter there service/arguments
+. on the agent: edit jenkins-agent.xml and alter there service/arguments
```
Comment (aHenryJard) [inline docs/troubleshooting.adoc:353]: are you sure that this property file exists ? `jenkins-agent.xml`

Comment (twasyl) [reply docs/troubleshooting.adoc:353]: I checked the source code and some other resources bundle, and it seems this file is already mentioned and searched

Comment (aHenryJard) [reply docs/troubleshooting.adoc:353]: ok then.

**Review by aHenryJard — APPROVED** [review]

**Review by amuniz — APPROVED** [review]
Looks good (ES translation specifically).

File: src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties [hunk 2]
```diff
@@ -22,6 +22,6 @@
 
 # Starts a Windows agent by <a href="http://en.wikipedia.org/wiki/Windows_Management_Instrumentation">a remote management facility</a> built into Windows. \
 # Suitable for managing Windows agents. \
-# Slaves need to be IP reachable from the master.
+# Agents need to be IP reachable from the controller.
 blurb=Iniciar um Windows agent <a href="http://en.wikipedia.org/wiki/Windows_Management_Instrumentation">uma ferramenta de gerenciamento remota</a> \
    agent precisam ter um IP alcan\u00e7\u00e1vel pelo master
```
Comment (rsandell) [inline src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties]: I don't speak Portuguese, but looks like you missed one.

File: src/main/resources/hudson/os/windows/Messages_es.properties [hunk 3]
```diff
@@ -21,9 +21,9 @@
 # THE SOFTWARE.
```
Comment (rsandell) [inline src/main/resources/hudson/os/windows/Messages_es.properties:21]: What about the actual `Messages.properties`?

File: src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties (hunk 2 above)
Comment (twasyl) [reply src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties]: You're right. Fixed

File: src/main/resources/hudson/os/windows/Messages_es.properties (hunk 3 above)
Comment (twasyl) [reply src/main/resources/hudson/os/windows/Messages_es.properties:21]: No changes were required in `Messages.properties` if it's what you mean :)

**Review by rsandell — APPROVED** [review]

**Comment by twasyl** [comment]
I don't have write access on the repo, so please don't wait for me to merge this one :) If someone could do it, would be awesome. Thanks!

**Review by MRamonLeon — APPROVED** [review]

**Comment by Saibamen** [comment]
This is the only change between previous version and 1.8: https://github.com/jenkinsci/windows-slaves-plugin/compare/windows-slaves-1.7...windows-slaves-1.8

Why did you release a new version without any bugfixes or dependency updates?

**Comment by aHenryJard** [comment]
> This is the only change between previous version and 1.8: [windows-slaves-1.7...windows-slaves-1.8](https://github.com/jenkinsci/windows-slaves-plugin/compare/windows-slaves-1.7...windows-slaves-1.8)
> 
> Why did you release a new version without any bugfixes or dependency updates?

Terminology changes are highly expected by several end users, you can either see that as a bug or a feature. In both cases a release is a good option. And anyway better release often and small no ?

**Pull request merged**